src/main/java/bio/terra/service/common/azure/StorageTableName.java

```diff
@@ -1,36 +1,14 @@
 package bio.terra.service.common.azure;
 
-import bio.terra.common.exception.FeatureNotImplementedException;
 import java.util.Objects;
 import java.util.UUID;
 
 public enum StorageTableName {
   SNAPSHOT("snapshot"),
   LOAD_HISTORY("loadHistory"),
   DEPENDENCIES("dependencies"),
-  DATASET("dataset") {
-    // TODO - With DR-2127, move remove special case for dataset
-    @Override
-    public String toTableName(UUID resourceId) {
-      throw new FeatureNotImplementedException(
-          "Dataset storage table names are not unique per resource. Use DATASET_TABLE.toTableName() instead.");
-    }
-
-    public String toTableName() {
-      return label;
-    }
-  },
-  FILES_TABLE("files") {
-    @Override
-    public String toTableName(UUID resourceId) {
-      throw new FeatureNotImplementedException(
-          "Files storage table names are not unique per resource. Use FILES_TABLE.toTableName() instead.");
-    }
-
-    public String toTableName() {
-      return label;
-    }
-  };
```

Comment:
Ok, I'm trying to jog my memory - We had these special cases for datasets and files because there would only be one dataset and file table due to one dataset/storage account. So, the name didn't have to be unique. 
... But, isn't that the case for all of these tables? Before this change, there would also only be one snapshot, load_history and/or dependency table per storage account since snapshots also always get their own storage account.

Reply:
Ah, so some of the tables were already doing the right thing (I think you were the one that actually started doing it first :) ) The dataset files and metadata tables were the only ones that weren't already doing the right thing and including their uuid

Reply:
Ah, ok, that sounds vaguely familiar. Thanks!